src/Files.App/UserControls/FilePreviews/MediaPreview.xaml.cs

```diff
@@ -31,6 +29,16 @@ private void PlayerContext_Loaded(object sender, RoutedEventArgs e)
 			ViewModel.TogglePlaybackRequested += TogglePlaybackRequestInvoked;
 		}
 
+		private void MediaPreview_Unloaded(object sender, RoutedEventArgs e)
+		{
+			// The MediaPlayerElement isn't properly disposed by Windows so we set the source to null
+			// to avoid issues the next time the control is used.
+			PlayerContext.Source = null;
+
+			PlayerContext.Loaded -= PlayerContext_Loaded;
+			Unloaded -= MediaPreview_Unloaded;		
```

Comment:
Could unloaded objects be reused? I don't know but if it is, this will cause the event to not work the next time it is loaded.

Reply:
It wouldn't hurt to double check but I'm pretty sure this isn't an issue.

Reply:
I can confirm that it's not an issue here.